.github/workflows/nightly-release.yml

```diff
@@ -26,7 +26,7 @@ defaults:
     shell: bash
 
 env:
-  RELEASE_BRANCH: "main"
```

Comment:
This doesn't need to be backported.  Only the workflow on `main` runs.

Reply:
Doesn't break anything though.

Reply:
was curious about that cause i had some more tests that are popping up in the alerts channel for `1.0` and `1.1` on `dbt-bigquery` failing for functional tests they shouldn't have didn't know if it was because of this or some other issue
https://dbt-labs.slack.com/archives/C02MC5GKCKX/p1686163397452839

Reply:
Because this is for a scheduled workflow, the workflow can only run from `main`.  That's why it won't break anything to backport, but will never make a difference.